lib/nerves_network.ex

```diff
@@ -68,6 +68,30 @@ defmodule Nerves.Network do
     :ok
   end
 
+  @doc """
+  Stops selected controls (aka managers) of `ifname`. The controls are being passed in a form of the keyword list.
+  Returns `{:ok, list()}`.
+
+  ## Parameters
+  - ifname: String identifying network interface's name i.e. "eth0"
+  - settings: a Keyword list with the settings i.e. [ipv4_address_method: :dhcp, ipv6_dhcp: stateless]. For stop function the key values are
+            irelevant because the settings are used for only locating an appropriate manager tied to the network interface specified with ifname.
+
+  ## Examples
+
+        iex> Nerves.Network.teardown("eth0", [ipv6_dhcp: :stateless])
+        {:ok, [ok: :ok]}
```

Comment:
Are you sure you want to return `{:ok, [ok: :ok]}` ?  I'm not sure how to interpret that. It might be best to just return `:ok` unless there is some other info you are trying to communicate